language/google/cloud/language/entity.py

```diff
@@ -84,7 +80,7 @@ class Entity(object):
     def __init__(self, name, entity_type, metadata, salience, mentions):
         self.name = name
         self.entity_type = entity_type
-        self.wikipedia_url = metadata.pop('wikipedia_url', None)
+        self.wikipedia_url = metadata.get('wikipedia_url', None)
```

Comment:
Right. But if that's the case, we should just get rid of `self.wikipedia_url`. The `pop` was there so it wasn't duplicated in the (always empty) `self.metadata` and in the `self.wikipedia_url`.